src/bin/sccache-dist/main.rs

```diff
@@ -44,6 +44,12 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Mutex, MutexGuard};
 use std::time::{Duration, Instant};
 
+#[cfg(any(
+    all(target_os = "linux", target_arch = "x86_64"),
+    target_os = "freebsd"
+))]
+use libc::{geteuid, uid_t};
```

Comment:
nit: I'd import it within the function, or use it's full path since it's the only place it's actually used.